src/util/message.h

```diff
@@ -275,55 +275,64 @@ class messaget
     return m;
   }
 
-  // in lieu of std::endl
-  static mstreamt &endl(mstreamt &m)
+  class commandt
```

Comment:
What's the value of this class over a `typedef unsigned commandt;`?

Reply:
Very different outcome -- we don't actually want to print that number!
Compare to STL's set_precision(...).

Reply:
Ah, of course, sorry for being dense. But maybe those not-so-smart people like me might benefit from a comment included in the code?

Reply:
done